src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -1249,6 +1256,794 @@ subroutine eigencalving(domain, err)
+         meanFlowParamA => meanFlowParamAField % array
+
+         !call mpas_pool_get_field(scratchPool, 'viscField',  viscField)
+         !call mpas_allocate_scratch_field(viscField, .true.)
+         !visc => viscField % array
+
+         call mpas_pool_get_field(scratchPool, 'sigmaxyField', sigmaxyField )
+         call mpas_allocate_scratch_field(sigmaxyField, .true.)
+         sigmaxy => sigmaxyField % array
+
+         call mpas_pool_get_field(scratchPool, 'sigmayxField', sigmayxField )
+         call mpas_allocate_scratch_field(sigmayxField, .true.)
+         sigmayx => sigmayxField % array
+
+         call mpas_pool_get_field(scratchPool, 'sigmaxxField', sigmaxxField )
+         call mpas_allocate_scratch_field(sigmaxxField, .true.)
+         sigmaxx => sigmaxxField % array
+
+         call mpas_pool_get_field(scratchPool, 'sigmayyField', sigmayyField )
+         call mpas_allocate_scratch_field(sigmayyField, .true.)
+         sigmayy => sigmayyField % array
+
+         call mpas_pool_get_field(scratchPool, 'sigmaminField', sigmaminField )
+         call mpas_allocate_scratch_field(sigmaminField, .true.)
+         sigmamin => sigmaminField % array
+
+         !call mpas_pool_get_field(scratchPool, 'sigmamaxField', sigmamaxField )
+         !call mpas_allocate_scratch_field(sigmamaxField, .true.)
+         !sigmamax => sigmamaxField % array
+
+         call mpas_pool_get_field(scratchPool, 'alphaField', alphaField )
+         call mpas_allocate_scratch_field(alphaField, .true.)
+         alpha => alphaField % array
+
+         !call mpas_pool_get_field(scratchPool, 'epsmaxField', epsmaxField )
+         !call mpas_allocate_scratch_field(epsmaxField, .true.)
+         !epsmax => epsmaxField % array
+
+         !call mpas_pool_get_field(scratchPool, 'nstarField', nstarField )
+         !call mpas_allocate_scratch_field(nstarField, .true.)
+         !nstar => nstarField % array
+
+         !call mpas_pool_get_field(scratchPool, 'szeroField', szeroField )
+         !call mpas_allocate_scratch_field(szeroField, .true.)
+         !szero => szeroField % array
+
+         !call mpas_pool_get_field(scratchPool, 'sourceField', sourceField )
+         !call mpas_allocate_scratch_field(sourceField, .true.)
+         !source => sourceField % array
+
+         !call mpas_pool_get_field(scratchPool, 'ddamagedtField', ddamagedtField )
+         !call mpas_allocate_scratch_field(ddamagedtField, .true.)
+         !ddamagedt => ddamagedtField % array
+
+         call mpas_pool_get_field(scratchPool, 'forceField', forceField )
+         call mpas_allocate_scratch_field(forceField, .true.)
+         force => forceField % array
+
+         call mpas_pool_get_field(scratchPool, 'meanTField', meanTField )
+         call mpas_allocate_scratch_field(meanTField, .true.)
+         meanT => meanTField % array
+
+         call mpas_pool_get_config(liConfigs, 'config_ice_density', config_ice_density)
```

Comment:
These two lines should be moved above before the block loop with the other config calls.  You can replace the last argument with rhoi, and then you don't need line 1676.  (And same for rhoo.)

Reply:
I moved these 2 lines up, but still keeps using config_ice_density (I changed all rhoi and rhoo to condig_ice/ocean_density)